### PR TITLE
Default values for line break mode and alignment

### PR DESCRIPTION
**Pull request by artemnovichkov**
By default `UILabel` uses `.natural` text alignment and `.byTruncatingTail` line break mode. To leave default behaviour as is I added these default values to all styles. Related to #13

### CODE REVIEW

File: Texstyle/TexstyleTests/LineAttributesTests.swift [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+final class LineAttributesTests: XCTestCase {
+
+    func testCopy() {
+        //Given
+        let attributes = LineAttributes.random
+        //When
+        let copy = attributes.copy()
+        //Then
+        XCTAssertEqual(copy, attributes, "Copy is not equal to line attributes")
```
Comment (ReshetovE) [inline Texstyle/TexstyleTests/LineAttributesTests.swift]: > "Copy is not equal to line attributes"

NOT equal?

File: Texstyle/TexstyleTests/TextStyleTests.swift [hunk 2]
```diff
@@ -90,4 +93,35 @@ final class TextStyleTests: XCTestCase {
...
     }
+
+    func testDefaultValues() {
+        //Given
+        let style = TextStyle()
+        //When
+
+        //Then
+        XCTAssertEqual(style.font, .systemFont(ofSize: 12), "Default style font must be equal to system font of 12 size")
+        XCTAssertEqual(style.color, .black, "Default color must be equal to nil")
```
Comment (ReshetovE) [inline Texstyle/TexstyleTests/TextStyleTests.swift]:
```suggestion
        XCTAssertEqual(style.color, .black, "Default color must be equal to black")
```

File: Texstyle/TexstyleTests/TextStyleTests.swift [hunk 3]
```diff
@@ -90,4 +93,35 @@ final class TextStyleTests: XCTestCase {
...
+        XCTAssertEqual(style.backgroundColor, .clear, "Default backgroundColor must be equal to nil")
```
Comment (ReshetovE) [inline Texstyle/TexstyleTests/TextStyleTests.swift]:
```suggestion
        XCTAssertEqual(style.backgroundColor, .clear, "Default backgroundColor must be equal to clear")
```

File: Texstyle/TexstyleTests/TextStyleTests.swift [hunk 4]
```diff
@@ -90,4 +93,35 @@ final class TextStyleTests: XCTestCase {
...
+        XCTAssertEqual(style.ligature, nil, "Default ligature must be equal to nil")
```
Comment (ReshetovE) [inline Texstyle/TexstyleTests/TextStyleTests.swift]:
```suggestion
        XCTAssertNil(style.ligature, "Default ligature must be equal to nil")
```

File: Texstyle/TexstyleTests/TextStyleTests.swift [hunk 5]
```diff
@@ -90,4 +93,35 @@ final class TextStyleTests: XCTestCase {
...
+        XCTAssertEqual(style.shadowStyle, nil, "Default shadowStyle must be equal to nil")
+        XCTAssertEqual(style.effectStyle, nil, "Default effectStyle must be equal to nil")
+        XCTAssertEqual(style.baselineOffset, nil, "Default baselineOffset must be equal to nil")
+        XCTAssertEqual(style.obliqueness, nil, "Default obliqueness must be equal to nil")
+        XCTAssertEqual(style.expansion, nil, "Default expansion must be equal to nil")
+        XCTAssertEqual(style.strikeThroughLine, nil, "Default strikeThroughLine must be equal to nil")
+        XCTAssertEqual(style.underlineLine, nil, "Default underlineLine must be equal to nil")
+        XCTAssertEqual(style.strokeColor, nil, "Default strokeColor must be equal to nil")
+        XCTAssertEqual(style.attachment, nil, "Default attachment must be equal to nil")
+        XCTAssertEqual(style.link, nil, "Default link must be equal to nil")
```
Comment (ReshetovE) [inline Texstyle/TexstyleTests/TextStyleTests.swift]: Here you can also replace `XCTAssertEqual` to `XCTAssertNil`

File: Texstyle/TexstyleTests/TextStyleTests.swift [hunk 6]
```diff
@@ -90,4 +93,35 @@ final class TextStyleTests: XCTestCase {
...
+        XCTAssertEqual(style.alignment, .natural, "Default alignment must be equal to natural")
+        XCTAssertEqual(style.lineBreakMode, .byTruncatingTail, "Default lineBreakMode must be equal to byTruncatingTail")
+        XCTAssertEqual(style.lineSpacing, nil, "Default lineSpacing must be equal to nil")
+        XCTAssertEqual(style.paragraphSpacing, nil, "Default paragraphSpacing must be equal to nil")
+        XCTAssertEqual(style.paragraphSpacingBefore, nil, "Default paragraphSpacingBefore must be equal to nil")
+        XCTAssertEqual(style.lineHeight, nil, "Default lineHeight must be equal to nil")
+        XCTAssertEqual(style.paragraphStyle, nil, "Default paragraphStyle must be equal to nil")
```
Comment (ReshetovE) [inline Texstyle/TexstyleTests/TextStyleTests.swift]: And here

File: Texstyle/TexstyleTests/LineAttributesTests.swift (hunk 1 above)
Comment (ntunin) [reply Texstyle/TexstyleTests/LineAttributesTests.swift]: I think so, this message contains a description what was wrong in this assertion

**Pull request merged**